fuse_models/include/fuse_models/common/sensor_proc.h

```diff
@@ -318,7 +321,16 @@ inline bool processAbsolutePoseWithCovariance(
   const auto indices = mergeIndices(position_indices, orientation_indices, position->size());
 
   populatePartialMeasurement(pose_mean, pose_covariance, indices, pose_mean_partial, pose_covariance_partial);
-  validatePartialMeasurement(pose_mean_partial, pose_covariance_partial);
+
+  try
+  {
+    validatePartialMeasurement(pose_mean_partial, pose_covariance_partial);
+  }
+  catch (const std::runtime_error& ex)
```

Comment:
I've added another commit that catches the `std::runtime_error` exceptions thrown by `validatePartialMeasurement` here and everywhere it's called.

This way, instead of letting the node running this crash because of the uncaught exception, a ROS ERROR message is printed, throttled every 10 seconds.

Reply:
@servos This is important because there case where the `pose_relative_covariance` obtained when `differential == true` and `independent == false` is sometimes slightly non-symmetric.

The precision for the symmetry validation check has also been relaxed to `1e-6` as you can see a little below.